README.md

```diff
@@ -1,24 +1,90 @@
-# README
+# hello_rails_react.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> A fullstack app with react and rails
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+- Ruby
+- Ruby on Rails v7.0.4
+- React
 
-* System dependencies
+## Getting Started
 
-* Configuration
+To get a local copy up and running follow these steps:
 
-* Database creation
+### Prerequisites
 
-* Database initialization
+- Ruby
+- Rails: 7.0.4
+- Postgres
+- Git
 
-* How to run the test suite
+### Usage
 
-* Services (job queues, cache servers, search engines, etc.)
+- Fork/Clone this project to your local machine
+- Open folder in your local enviroment and run thes lines of code to get started:
 
-* Deployment instructions
+Install gems with:
 
-* ...
+```Ruby
+    bundle install
+```
+
```

Comment:
### {OPTIONAL}

- It seems I had to run `yarn install ` for the application to run. It is also great to add this to the readme file to ensure that the user of your application knows this.